src/routes/safe/components/Apps/components/ThirdPartyCookiesWarning.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import MuiAlert from '@material-ui/lab/Alert'
+import MuiAlertTitle from '@material-ui/lab/AlertTitle'
+import { Link } from '@gnosis.pm/safe-react-components'
+
+type Props = {
+  onClose: () => void
+}
+
+const HELP_LINK =
+  'https://help.gnosis-safe.io/en/articles/5955031-why-do-i-need-to-enable-third-party-cookies-for-safe-apps'
+
+export const ThirdPartyCookiesWarning = ({ onClose }: Props): React.ReactElement => {
+  return (
+    <MuiAlert severity="warning" onClose={onClose}>
+      <MuiAlertTitle>
+        Third party cookies disabled. Safe apps may not work properly under this circumstances. You can find the
+        information about this problem in{' '}
+        <Link href={HELP_LINK} size="xl" target="_blank">
+          Help Article
+        </Link>
```

Comment:
I'd rephrase this a little:

```ts
Third party cookies are disabled. Safe Apps may therefore not work properly. You can find out
more information about this{' '}
<Link href={HELP_LINK} size="xl" target="_blank">
  here
</Link>.
```

Reply:
In this case I would ping @liliiaorlenko because this is the text from [the design](https://www.figma.com/file/CSMM622zKmfeMs2DUPOILh/Safe-Apps-issue-with-third-party-cookies?node-id=19%3A57)

Reply:
if its better wording pls change it :)